src/Bonsai.Ephys.Design/WaveformVisualizer.cs

```diff
@@ -322,6 +323,16 @@ unsafe void MenuWidgets()
                     ImGui.EndTable();
                 }
 
+                ImGui.TableNextColumn();
+                var color = invertSignal ? ImGui.GetColorU32(ImGuiCol.ButtonActive) : ImGui.GetColorU32(ImGuiCol.Button);
+                ImGui.PushStyleColor(ImGuiCol.Button, color);
+                if (ImGui.Button("Invert\nSignal"u8, buttonSize))
+                {
+                    invertSignal = !invertSignal;
+                }
+
+                ImGui.PopStyleColor();
```

Comment:
```suggestion
                ImGui.TableNextColumn();
                var color = invert ? ImGui.GetColorU32(ImGuiCol.ButtonActive) : ImGui.GetColorU32(ImGuiCol.Button);
                ImGui.PushStyleColor(ImGuiCol.Button, color);
                if (ImGui.Button("Invert"u8, buttonSize))
                {
                    invert = !invert;
                }

                ImGui.PopStyleColor();
```

I was also wondering it might be preferable UI-wise to have this button sit next to the `Pause` button since it feels functionally more similar than the color themes (this would be moving the code upward a bit).